tests/test_pickling.py

```diff
@@ -9,18 +9,41 @@
 from pybind11_tests import pickling as m
 
 
-def test_pickle_simple_callable():
+def test_assumptions():
+    assert pickle.HIGHEST_PROTOCOL >= 0
```

Comment:
Why this? This isn't a test for pybind11, it's a test of the Python standard library? https://docs.python.org/3/library/pickle.html#pickle.HIGHEST_PROTOCOL and it's at least 4 since Python 3.4.